docs/gherkin.md

```diff
@@ -0,0 +1,112 @@
+# Gherkin
+
+Gherkin is the language that Cucumber understands. It is a Business Readable, [Domain Specific Language](http://martinfowler.com/bliki/BusinessReadableDSL.html) that lets you describe software’s behaviour without detailing how that behaviour is implemented.
+
+Gherkin serves two purposes — documentation and automated tests. The third is a bonus feature — when it yells in red it’s talking to you, telling you what code you should write.
+
+Gherkin’s grammar is defined in the Treetop grammar that is part of the Cucumber codebase. The grammar exists in different flavours for many spoken languages (37 at the time of writing), so that your team can use the keywords in your own language.
```

Comment:
This is pretty technical for an introduction. I think all we should mention is that a lot of spoken languages (much more than 37 nowadays I believe) are supported. Keeping the part about why this matters (i.e. use the language of your team/project) is good though.